### ISSUE TITLE
Regression in Table "+/- Filter buttons"

### ISSUE THREAD

**Issue by Ultraseamus** [grafana/grafana#30660]
**What happened**:

The "+/-" filter buttons stopped showing up in the Table cells.

**What you expected to happen**:

For them to show up as they did in previous versions.

**How to reproduce it (as minimally and precisely as possible)**:

Create a table with at least 1 cell that contains filterable data, hover over that cell.

**Anything else we need to know?**:

At first I thought this feature was intentionally removed, but looking further at the code (and fixing it in a local build), I think it may have just been an oversight with the latest large Table update.

In the styles file, the display value for the filter buttons was set to "none", and the logic to show/hide the filters based on mouse enter/mouse leave was left out. I have a fix for this (which is pretty simple, admittedly) that I could share, if that would be welcome.

If this was left out intentionally, I would suggest adding a control to turn it back on, or maybe a feature selection in the config. It appears to be working fine after I made the changes described above, and it is a valuable feature.

**Environment**:
- Grafana version: 7.3.6

**Comment by ivanahuckova** [comment]
Hello @Ultraseamus, filtering in table has change in Grafana 7.2. See [this documentation](https://grafana.com/docs/grafana/latest/panels/visualizations/table/filter-table-columns/) for more details on how to use it.

**Comment by Ultraseamus** [comment]
But that is a filter local to the table, it's not the same thing.

I am talking about he filter that sets a Dashboard-wide adhoc filter.

Also, the support for the filter I am referring to is still present in the code, and it is fully compatible with the react table filter you are referring to.

If the intention is to deprecate the adhoc filter buttons, I think that is a mistake.

**Comment by torkelo** [comment]
the ad hoc filters are working in 7.3.6 with Elasticsearch, https://play.grafana.org/d/000000014/elasticsearch-metrics?orgId=1&var-NewFilter= 

what data source are you using?

**Comment by Ultraseamus** [comment]
@torkelo: Huh... interesting. My datasource is elasticsearch. I notice that your dashboard is running on 7.4, but looking at the code it made sense to me that this feature would not work in either version. So I've clearly made a mistake.

Would you be able to point me to the file where the filterWrapper display property is set off of "none"? I thought I'd find it in "FilterActions" (I will say that the term "filter" is a bit overloaded at this point :)). Maybe there is some condition I am missing? I know that when I run it, the display for those elements is set to "none" and I can't get them to show up without changes.

Thanks!

**Comment by torkelo** [comment]
can you show the inspector drawer / JSON tab / DataFrame structure. 

does the field config have filterable: true property? 

we have no other reports that this is  not working in 7.3, so it's very strange

**Comment by Ultraseamus** [comment]
This ended up being my issue, thanks for pointing me in the correct direction. :)

I don't know how it happened, but at some point

**Comment by Ultraseamus** [comment]
This ended up being my issue, thanks for pointing me in the right direction. :)

This is likely not going to be useful for you, but if you were curious:

I don't know how it happened, but in the "styles" file a non-ascii character was hiding from me on the line where ".cell-filter-actions" was defined. And that character became part of the class name. My text editors were not showing it, and my diff tools did not detect it as a difference. I found it by opening up Dev Tools and adding a class to the element, autocomplete prompted me with the name with some gibberish on the end. So I toggled over to UTF-8, and the character showed up.

Thanks again, sorry for the false alarm.
